gc/base/standard/CompactScheme.cpp

```diff
@@ -339,6 +339,30 @@ MM_CompactScheme::setFreeChunk(omrobjectptr_t from, omrobjectptr_t to)
 	return size;
 }
 
+MMINLINE void
+MM_CompactScheme::preObjectMove(MM_EnvironmentBase *env, omrobjectptr_t objectPtr)
+{
+	env->preMove(_extensions, objectPtr);
+}
+
+MMINLINE void
+MM_CompactScheme::postObjectMove(MM_EnvironmentBase *env, omrobjectptr_t objectPtr)
+{
+	env->postMove(_extensions, objectPtr);
+
+#if defined(OMR_ENV_DATA64)
```

Comment:
To make slightly more readable here (and few other similar spots), we could put these #ifdef inside fixupObjectDataAddr. 
Since fixupObjectDataAddr is inlined, compiler _should_ be able to see that it's dead code for 32bit, and remove isIndexable check, as well.
Up to you, what you prefer more (or how much you trust/care compiler to optimize 32bit).

Reply:
That's true, I was thinking that if we put the #ifdef inside fixupObjectDataAddr, it would improve readability here and worsen there; but now taking a closer look, it's probably worth making the trade since fixupObjectDataAddr is the external dataAddr fixup API call which is used in quite a few places